org.musescore.MuseScore.yml

```diff
@@ -44,49 +45,33 @@ cleanup-commands:
 modules:
   - shared-modules/lame/lame-3.99.5.json
 
-  - name: portaudio
-    config-opts:
-      - --disable-static
-      - --without-oss
-    sources:
-      - type: archive
-        url: http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz
-        sha256: f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
+  - name: jack2
+    buildsystem: simple
+    build-commands:
+      - python3 waf configure --prefix=/app --htmldir=/app/share/doc/jack/ --freebob=no --classic
+      - python3 waf build
+      - python3 waf install
     cleanup:
-      - /lib/pkgconfig
+      - /bin
       - /include
-      - '*.la'
-
-
-  - name: portmidi
-    buildsystem: cmake-ninja
-    config-opts:
-      - -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/app/lib
-      - -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY=/app/lib
-      - -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=/app/bin
-    post-install:
-      # MuseScore looks for this name for some reason
-      - ln -s libportmidi.so /app/lib/libporttime.so
+      - /share
+      - /lib/jack
+      - /lib/libjackserver.so*
+      - /lib/pkgconfig
     sources:
       - type: archive
-        url: https://sourceforge.net/projects/portmedia/files/portmidi/217/portmidi-src-217.zip
-        sha256: 08e9a892bd80bdb1115213fb72dc29a7bf2ff108b378180586aa65f3cfd42e0f
-      - type: patch
-        path: portmidi-no-java.patch
-    cleanup:
-      - /bin
-      - /lib/pkgconfig
-      - /include
-      - '*.a'
-      - '*.la'
-
+        url: https://github.com/jackaudio/jack2/archive/v1.9.12.tar.gz
+        sha512: f0271dfc8f8e2f2489ca52f431ad4fa420665816d6c67a01a76da1d4b5ae91f6dad8c4e3309ec5e0c159c9d312ed56021ab323d74bce828ace26f1b8d477ddfa
 
   - name: musescore
     buildsystem: cmake
     config-opts:
       - -DCMAKE_BUILD_TYPE=RELEASE
+      - -DBUILD_PORTAUDIO=OFF
+      - -DBUILD_PORTMIDI=OFF
       - -DBUILD_ALSA=OFF
-      - -DBUILD_JACK=OFF
+      - -DBUILD_JACK=ON
+      - -DUSE_SYSTEM_FREETYPE=ON
```

Comment:
Don’t combine random stuff into one commit/PR. There was a time when I used this flag, but really the font rendering is much worse without it, so I ultimately decided against it.